docs/devel/storage.md

```diff
@@ -31,7 +36,55 @@ This will create
 
 After a few moments the hostpath-provisioner pod should be running, we are now ready to create PVCs against it. Any requested PVCs will be stored in `/var/run/kubevirt/hostpath/<pvc-name>-<hash>`. The PV retention policy will be `delete`, which means that when you delete the PVC, the matching PV is also removed and the directory created on the host is removed as well.
 
-## Deploy CDI controller
+# Local volume based storage
+
+Local volume is in Beta since Kubernetes 1.10 and thus no longer requires you to enable it through the [feature-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) parameter of the kubelet. To enable local volume for the development environment please follow the following steps. Note local volumes WILL work on multiple nodes due to the fact that PVCs don't get bound until a POD actually uses the PVC, at which point the local volume is bound on the node the pod is running on. The following steps are based on information found in the [Local Persistent Storage User Guide](https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/README.md)
+
+## Pre-liminary steps
+
+Since you can't add disks or volumes after a container is started (your nodes are running in containers in the development environment) you will have to use some directories as the mount source. This can be achieved by using the
```

Comment:
I think in general it's better to avoid words such as `I` `my` or `you`. I don't know if there's any rule against them, but it's generally not seen in these sort of documents.

Reply:
Rephrased a lot of that to not include those words. I was trying to make a friendly step by step guide. But you a right this is more dry technical stuff. Thank you for the review.